hs_app_netCDF/models.py

```diff
@@ -8,14 +7,18 @@
 from mezzanine.pages.page_processors import processor_for
 
 from dominate.tags import legend, table, tbody, tr, td, th, h4, div, strong, form, button, _input
+from rdflib import RDF, BNode, Literal
+from rdflib.namespace import DCTERMS
 
+from hs_core.hs_rdf import HSTERMS, rdf_terms
 from hs_core.models import BaseResource, ResourceManager
 from hs_core.models import resource_processor, CoreMetaData, AbstractMetaDataElement
 from hs_core.hydroshare.utils import get_resource_file_name_and_extension, \
-    add_metadata_element_to_xml, get_resource_files_by_extension
+    get_resource_files_by_extension
 
 
 # Define original spatial coverage metadata info
+@rdf_terms(HSTERMS.spatialReference)
```

Comment:
@sblack-usu I am wondering now that we don't have netCDF resources, but only have netCDF aggregations in composite resources, is this hs_app_netCDF used anywhere in our codebase?

Reply:
It is still used.  A lot of the logic for resources were just adapted to the aggregation model.